### ISSUE TITLE
Designing examples: Ensure interactive elements are easy to identify

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-quick-start#76]
http://w3c.github.io/wai-quick-start/designing.html#ensure-interactive-elements-are-easy-to-identify

1. I really do not like those colors. In the first text block: I thought we weren't going to use anything like the WAI link color? In the links examples: the magenta/pink is so bright. Can you just use common browser default colors for links? [medium]
2. For "Mouse hovered over link" I though EOWG suggested a pale highlight (I think in Brent's comments &/or mine &/or in the telecon)? [medium-low]
3. Why would you have different visual treatment for mouse hover verses keyboard focus? [medium]
4. Personally I do not like it when things become reverse on hover or touch or other -- as you have for "Touch or click on link". However, if others feel strongly, I'll bow to the preference of the group. [low-medium]

**Comment by iadawn** [comment]
1. The challenge is that to find a redish color that meets AAA, it will be like WAI style. I can try to go to common browser colors, but my [moderate] concern with that is that it is not really showing a 'designed' set of link colors.
1. It was suggested, I am a little bit hesitant as it is not really a common approach to hover. The tendency that I encounter is a color change and an underline... or sometimes just an underline. I can put one in to show it.
1. I would make :focus more visible to make it really obvious for keyboard users. I have made it different in this case to highlight that this can be done. The alternative is really to just have one style for :hover and :focus. Happy for your thoughts on my reasoning.
1. Again, this is a more common approach to activation styling.

What I have tried to do with these styling choices is to make a consistent standalone style. I am hesitant to drop to standard styling (for reasons above), I am also trying to create quite different colors for the plain and other styles, and this is all suffering from it being on a grey background and needing to be AAA.
